filebeat/registrar/registrar.go

```diff
@@ -34,12 +38,13 @@ var (
 	registryWrites = monitoring.NewInt(nil, "registrar.writes")
 )
 
-func New(registryFile string, out successLogger) (*Registrar, error) {
+func New(registryFile string, flushTimeout time.Duration, out successLogger) (*Registrar, error) {
```

Comment:
Should we call the config also flush_timeout? Would make it more explicit.

Reply:
you mean `filebeat.registry_flush_timeout`?